### PR TITLE
fixed fileupdater crash for string routine entries

### PR DESCRIPTION
**Pull request by ArnoldSmith86**
When routines contain entries that are just a string (like the new dynamic expressions), the fileupdater for v3 would crash.

This PR makes it skip any entries that do not have a `func` parameter.

### CODE REVIEW

**Comment by ArnoldSmith86** [comment]
PR-SERVER-BOT: You can play around with it here: http://212.47.248.129:3583/pr-test (or any other room on that server)

File: server/fileupdater.mjs [hunk 1]
```diff
@@ -157,6 +157,9 @@ function v3RemoveComputeAndRandomAndApplyVariables(routine) {
   }
 
   for(const [ i, op ] of Object.entries(routine)) {
+    if(!op || !op.func)
```
Comment (robartsd) [inline server/fileupdater.mjs:160]: What is !op accomplishing here?

Comment (ArnoldSmith86) [reply server/fileupdater.mjs:160]: If `op == null`, it would crash otherwise.

**Review by robartsd — APPROVED** [review]
Code looks okay. Not sure how to go about testing.

**Pull request merged**